sympy/polys/polyutils.py

```diff
@@ -420,6 +437,28 @@ def _dict_reorder(rep, gens, new_gens):
 
     return map(tuple, new_monoms), coeffs
 
+def has_imaginary(exprs):
+    """Checks whether the numerator, denominator of an expression has Imaginary Unit"""
```

Comment:
This is not telling what form `exprs` is in. It says "expression" but then you check for `len(exprs)` below which suggests that it is an iterable.